deepC/dnnc.api

```diff
@@ -860,20 +861,36 @@ tensor<output> cosh(tensor<input> &a){
   }
 }
 
-tensor<output> slice(tensor<output> &a, 
-	                 tensor<input> &start, 
-		             tensor<input> &end, 
-		             tensor<input> axes = NULL_TENSOR<input>, 
-		             tensor<input> steps = NULL_TENSOR<input>) {
-	Slice<output, output, input> op;
+tensor<output> slice(tensor<input> &a,
```

Comment:
Please change it back to last definition. Let me know, if you want to know why?

Also make slice and set_slice consistent.